Directory.Build.props

```diff
@@ -4,18 +4,18 @@
     <Authors>Workleap</Authors>
     <Owners>Workleap</Owners>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <LangVersion>10</LangVersion>
+    <LangVersion>12</LangVersion>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <Description>Provides MongoDB access through .NET dependency injection, following Microsoft.Extensions.* library practices.</Description>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Workleap.DotNet.CodingStandards" Version="1.1.4">
+    <PackageReference Include="Workleap.DotNet.CodingStandards">
```

Comment:
No need for this one. You can add the following to Directory.Pakcages.props (works for any dev dependencies)

````
<GlobalPackageReference Include="Workleap.DotNet.CodingStandards" Version="1.1.4" />
````

Reply:
So, you prefer keeping it this way instead of in the package file :(